controllers/comments.controller.js

```diff
@@ -0,0 +1,16 @@
+const {fetchCommentsByReview} = require("../models/comments.model.js")
+const {fetchReview} = require("../models/reviews.model.js")
+
+exports.getCommentsByReview = (req, res, next) => {
+    fetchReview(req.params.id)
```

Comment:
Just be careful with running multiple async functions, as we need to ensure that the response is only sent once both are successfully run. Can you chain your .thens to ensure that the response is only sent once everything has been run? Alternatively, you may want to look at Promise.all, if that's a more comfortable syntax for you